### PR TITLE
More Symfony 4 updates for the service tags article

### PR DESCRIPTION
**Pull request by javiereguiluz**

### CODE REVIEW

**Review by weaverryan — APPROVED** [review]

**Comment by weaverryan** [comment]
Status: Reviewed

File: service_container/tags.rst [hunk 1]
```diff
@@ -245,18 +251,18 @@ Register the Pass with the Container
...
-    // src/AppBundle.php
-
-    // ...
-    use Symfony\Component\DependencyInjection\ContainerBuilder;
+    // src/Kernel.php
     use App\DependencyInjection\Compiler\MailTransportPass;
+    // ...
 
-    class AppBundle extends Bundle
+    class Kernel extends Kernel
```
Comment (yceruto) [inline service_container/tags.rst]: `namespace App;` + `use ... as BaseKernel` + `BaseKernel`?

**Comment by xabbuh** [comment]
Thank you Javier.

File: service_container/tags.rst (hunk 1 above)
Comment (xabbuh) [reply service_container/tags.rst]: fixed in 1a3be4386f60f9c2e88995406bf82b12397d8759

**Pull request merged**